content/overview/getting-started/_index.en.md

```diff
@@ -5,50 +5,149 @@ weight: 2
-enable = false
+#### Linking Actors and Capability Providers
+Now your `Echo` Actor and `HTTP Server` Providers are running, but they aren't connected. In order to allow the Actor and Provider to communicate, we need to `link` them together, which we do using their respective Public Keys (shown in the Inventory as `Actor ID` and `Provider ID`).
 
-####### Default parameters ###########
-[params]
-logo = "images/logo.png"
-# Meta data
-description = "This is meta description"
-author = "Themefisher"
-# contact form action
-contact_form_action = "#" # contact form works with https://formspree.io
-# Google Analitycs
-google_analitycs_id = "" # your id
+You can use the following command to `link` your Actor and Provider, on MacOS you might receive a message asking if you want `wash` to receive incoming network connections. You should click "Allow", this is just setting up the provider's local HTTP server to listen on port 8080.
+```shell
+wash> ctl link MBCFOPM6JW2APJLXJD3Z5O4CN7CPYJ2B4FTKLJUR5YR5MITIU7HD3WD5 VAG3QITQQ2ODAOWB5TTQSDJ53XK3SHBEIFNK4AYJ5RKAX2UNSCAPHA5M wasmcloud:httpserver PORT=8080
+```
+Once you see that the link has been advertised, you are ready to send a request to your actor.
 
-```
\ No newline at end of file
+#### Interacting with your Actor
+In another terminal window (not the REPL), run the following command:
+```shell
+curl localhost:8080/echo
+```
+In response, you should receive your request object (notice the path argument):
+```shell
+{"method":"GET","path":"/echo","query_string":"","headers":{"host":"localhost:8080","user-agent":"curl/7.64.1","accept":"*/*"},"body":[]}
+```
+Feel free to try out different methods of making a request to your Actor, including adding headers or using a different HTTP method to see different outputs.
+
+Instead of using `curl`, you can also _directly invoke_ actors' registered functions using `ctl call`. As mentioned before, the function that "echoes" this HTTP request is registered with the operation `HandleRequest`, and we can make this request directly to the actor if we supply the correct parameters.
+
+Here's an example of using `call` in the REPL to mimic our above `curl`. Note that this is not interacting with the HTTP server provider, and we don't need it to be running or linked for this operation to succeed:
+```shell
+wash> ctl call MBCFOPM6JW2APJLXJD3Z5O4CN7CPYJ2B4FTKLJUR5YR5MITIU7HD3WD5 HandleRequest {"method": "GET", "path": "/echo", "body": "", "queryString":"","header":{}}
+```
+Our output will look something like this:
+```shell
+Call response (raw): ��statusCode�Ȧstatus�OK�header��body�S{"method":"GET","path":"/echo","query_string":"","headers":{},"body":[]}
+```
+Because of our method of serialization / deserialization, the return payload has some characters that the terminal doesn't know how to interpret from its bytes.[<sup>1</sup>](#references) However you can still see the response body which contains our exact "echoed" request.

```

Comment:
maybe change "Because of our method of serialization  / deserialization" we can just say "Because the actor isn't actually returning JSON"...